### PR TITLE
clarify PULL_REQUST_TEMPLATE.md & CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Mikaela**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 ## Description
 
-Resolves: #none <!-- The number of the issue that is resolved by this pull request. If there is none, feel free to delete this line -->
+Resolves: #none <!-- The number of the issue that is resolved by this pull request. This should be number of the discussion issue. If there is none, feel free to delete this line -->
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]: First hint that we do want the discussion issue to exist.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 
 #### Check List <!-- Please add an x in each box below, like so: [x] -->
 
 - [ ] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).
 
-- [ ] I have [listed the source code](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).
+- [ ] I have listed the source code in [source_code.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think this is more clear formatting.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,18 +2,18 @@
...
 
-- [ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).
+- [ ] The project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]: I may want to check that wiki page and if there is a better one.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -2,18 +2,18 @@
...
 
-- [ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).
+- [ ] The project has an [associated discussion]: # <!-- issue number -->
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] The project has an associated discussion: # <!-- issue number -->
```

oops. Anyway the previous formatting was very unhelpful in my opinion as it just went to issue page and forced me to use search as no one (me included) never changed the link.

and I think `The` fits here better than `This`, even if there is probably no practical difference.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -2,18 +2,18 @@
...
 
 * Netlify preview for the mainly edited page: <!-- link or Non Applicable? Edit this in afterwards -->
 
-* Code Repository (if applicable): 
+* Code repository of the project (if applicable):
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]: We recently had a PR that listed the fork here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -2,18 +2,18 @@
```
Comment (Mikaela) [inline .github/PULL_REQUEST_TEMPLATE.md]: Oh and I wonder if we should also mention CODE_OF_CONDUCT.md on the top, but if we get a project that is full of hate, I guess we can refer to it separately.

**Comment by Mikaela** [comment]
Open source and free software are very overlapping, but open source is more inclusive and I think it's what we actually care about here instead of free software movement.

Example case: I remembered @blacklight447-ptio and @dawidpotocki discussing VeraCrypt recently as VeraCrypt is not free software, but is open source software.

* https://en.wikipedia.org/wiki/VeraCrypt#License_and_source_model

**Comment by dawidpotocki** [comment]
I was discussing that TrueCrypt (from which VeraCrypt is forked) IS NOT Open Source.

> but VeraCrypt inherited a substantial amount of code from its predecessor TrueCrypt, and also inherited the __source-available__ TrueCrypt License for those files.

It's not Open Source. Please read definition at https://opensource.org/osd and then read TrueCrypt license. For bonus points you can also read Free Software definition at https://www.gnu.org/philosophy/free-sw.html.

> You may make copies of This Product
(unmodified) and distribute copies of This Product (unmodified)
that are not included in another product forming Your Product
(except as permitted under Chapter III). 

First thing in license and it already makes it not open source

> Discussion of the licensing terms on the Open Source Initiative (OSI)'s license-discuss mailing list in October 2013 suggests that the TrueCrypt License has made progress towards compliance with the Open Source Definition but would not yet pass if proposed for certification as Open Source software.

According to current OSI president Simon Phipps:

>    ...it is not at all appropriate for [TrueCrypt] to describe itself as "open source." This use of the term "open source" to describe something under a license that's not only unapproved by OSI but known to be subject to issues is unacceptable.

From https://en.m.wikipedia.org/wiki/TrueCrypt

**Comment by Mikaela** [comment]
Do I understand correctly that that leaves two choices where to go with this PR?

* Require source to be available (and where to link to explain it)?
* Mentally note that "open source" is not a trademark and ignore OSI's preference as long as we won't start calling things "OSI certified"?

> The term "open source" was originally intended to be trademarkable; however, the term was deemed too descriptive, so no trademark exists.[80] The OSI would prefer that people treat open source as if it were a trademark, and use it only to describe software licensed under an OSI approved license.[81]
> OSI Certified is a trademark licensed only to people who are distributing software licensed under a license listed on the Open Source Initiative's list.[82]

* https://en.wikipedia.org/wiki/Open-source_software#Free_software

* * * * *

And what do you think of the PR itself?

**Comment by Mikaela** [comment]
Wait, is the case of VeraCrypt even relevant to this PR other than my comment being incorrect in case of OSIs preference being heard? Is it just free software vs open source software? How about Free Libre Open Source Software?

**Comment by blacklight447-ptio** [comment]
Tbh, if you can read the source code, I call it open source, no matter what license.

**Comment by ghbjklhv1** [comment]
@Mikaela I think everybody at this point in time knows what free/libre software is.

**Comment by ghbjklhv1** [comment]
> Open source and free software are very overlapping, but open source is more inclusive and I think it's what we actually care about here instead of free software movement.

I agree with [@blacklight447-ptio](https://github.com/privacytoolsIO/privacytools.io/pull/1291#issuecomment-530150637) that open-source is too broad and doesn't get to the point of being able to read the software and remix the software.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (nitrohorse) [reply .github/PULL_REQUEST_TEMPLATE.md]: Hmm, I'm not sure but it doesn't hurt to add it as a comment at the top as a reminder.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (nitrohorse) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would probably tidy this up a bit to:
```
<!-- A link to the discussion issue(s) resolved by this pull request. If there is none, feel free to delete this line. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -2,18 +2,18 @@
...
 - [ ] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).
 
-- [ ] I have [listed the source code](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).
+- [ ] I have listed the source code in [source_code.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).
 
-- [ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).
+- [ ] The project is [open source software](https://en.wikipedia.org/wiki/Open-source_software).
 
-- [ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).
+- [ ] The project has an associated discussion: # <!-- issue number -->
```
Comment (nitrohorse) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this somewhat redundant due to already having a "Resolves"? Could we remove this line in favor of having the "Resolves"?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (Mikaela) [reply .github/PULL_REQUEST_TEMPLATE.md]: :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Mikaela) [reply .github/PULL_REQUEST_TEMPLATE.md]: :+1: I think I also need to clarify that discussion issue means software suggestion to distinguish it from discussions about news which we hoped to move to the forums.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (Mikaela) [reply .github/PULL_REQUEST_TEMPLATE.md]: I guess

**Comment by Mikaela** [comment]
@privacytoolsIO/editorial What are your thoughts on the Open Source Software vs Free/Libre Software?

**Comment by nitrohorse** [comment]
> @privacytoolsIO/editorial What are your thoughts on the Open Source Software vs Free/Libre Software?

So currently the template states:
> [ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).

While the suggested change is:
> [ ] The project is [open source software](https://en.wikipedia.org/wiki/Open-source_software).

From my observation of ptio, it seems like we aim for the software ptio suggests to be free/libre but also accept software that isn't "free" (apologies, I don't have examples at the moment, but I'm sure there are throughout the site). So the free/libre aspect seems to be our goal, not a requirement. If this is true, then the suggested change makes sense to make and make we could expand it to something like this?
```
[ ] The project is [open source software](https://en.wikipedia.org/wiki/Open-source_software). <!-- Even better if it's free (https://www.gnu.org/philosophy/free-sw.html) / libre software (https://en.wikipedia.org/wiki/Free_software)! -->
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -7,6 +7,9 @@ Please read this before contributing.
 - Be nice and respectful.
 - English only.
 - Be constructive.
+- Please feel free to *review changes* in the *files changed* tab of any
+  pull request at any time.
+- See also our [Code of Conduct](https://github.com/privacytoolsIO/privacytools.io/blob/master/CODE_OF_CONDUCT.md)
```
[the remark this answers is not in the document]
Comment (Mikaela) [reply .github/CONTRIBUTING.md]: Sorry, could you clarify? It's good to add a link to the CoC as a reminder here, or should English-only be documented somewhere else?

I kind of would like to have language specific subforums and accept contributions or at least feedback in other languages too, assuming we have team member(s) fluent in that language.

[the remark this answers is not in the document]
Comment (nitrohorse) [reply .github/CONTRIBUTING.md]: Sorry, I meant to say I think we only have this English only version of the CoC at the moment so adding a link to it like your change does I think is good 👍🏼

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -37,24 +40,24 @@ Note: This criteria applies to all of privacytools.io
...
 ### Hardware
-- Must be [H-Node Class A](https://h-node.org/wiki/page/en/compatibility-classes) or Equivlant (if applicable)
+- Must be [H-Node Class A](https://h-node.org/wiki/page/en/compatibility-classes) or Equivalant (if applicable)
 - Must prioritize hardware certifications like [RYF](https://ryf.fsf.org/), [OSHWA](https://certification.oshwa.org/), and OSI when avalible.
 - Cannot lock users to a particular platform.
 
 ### Software
 - Must be able to download over encrypted network (can be a mirror)
-- Must be free software
+- Must be Open Source Software
```
[the remark this answers is not in the document]
Comment (nitrohorse) [reply .github/CONTRIBUTING.md]: Yeah, I would this so for consistency

**Pull request merged**